person_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'nameable'
+require 'securerandom'
+require_relative 'person'
+
+# test class person
+describe Person do
+  describe '#new' do
+    context 'when given valid parameters' do
+      # test
+      it 'returns a person object' do
+        # test code
+        person = Person.new(18, 'John Doe')
+        expect(person).to be_an_instance_of(Person)
+      end
+    end
+  end
+end
```

Comment:
- [ ] Following testing best practices, Kindly create a folder in your project and name it "spec" and put all of the test files inside it. This would make your testing more professional and ease the process of running your tests, as you can run `rspec spec` to run all of the tests inside the `spec` folder.
- [ ] Please make sure to implement more tests for each class.

- [HINT] As an example for the `Person` class, you can test the following functionalities:
   - Having the "Unknown" name value if the user didn't pass in a name.
   - Testing the `can_use_services?` method while passing in different values to different person objects.
   - Testing the `add_rental` method while adding a new rental.

Reply:
@ShadyShawkat  Thank you for reviewing my project, I will make the necessary corrections as suggested